source/sphinx_extensions/plugin_directory/extension.py

```diff
@@ -57,6 +57,7 @@ def generate_rst(app):
             write_bibtex(plugin.citations, index_bib)
 
         for action in plugin.actions.values():
+            action_api_name = action.id
```

Comment:
This is super nitpicky, but maybe this makes more sense inlined into the context definition, [below](https://github.com/qiime2/docs/pull/326/files#diff-e380915e2a96fdaf7dd9aa48d1fb50d8R93) (like you did for `plugin_api_name`). Functionally there is no difference, but maybe keeping things consistent will help future readers of this code...